python/sqlflow_submitter/explainer.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019 The SQLFlow Authors. All rights reserved.
```

Comment:
Maybe we can rename this file to `plotter.py`

Reply:
I plan to add some common explanation code in this file, `explainer.py` would be more appropriate.